sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/AboutDirectorySteps.java

```diff
@@ -0,0 +1,160 @@
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.testng.asserts.SoftAssert;
+
+public class AboutDirectorySteps implements En {
+  public static final String userDirPath = System.getProperty("user.dir");
+  public static final List<String> xlsxFileContentList = new ArrayList<>();
+  public static String language;
+
+  @Inject
+  public AboutDirectorySteps(WebDriverHelpers webDriverHelpers, SoftAssert softly) {
+
+    When(
+        "I select {string} language from Combobox in User settings",
+        (String chosenLanguage) -> {
+          language = chosenLanguage;
+          webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, chosenLanguage);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+        });
+
+    When(
+        "I set on default language as English in User settings",
+        () -> {
+          String defaultLanguage = "";
+          switch (language) {
+            case "Fran\u00E7ais":
+              defaultLanguage = "Anglais";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+            case "Fran\u00E7ais (Suisse)":
+              defaultLanguage = "Anglais";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+            case "Deutsch":
+              defaultLanguage = "English";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+            case "Deutsch (Schweiz)":
+              defaultLanguage = "English";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+            case "Espa\u00F1ol (Ecuador)":
+              defaultLanguage = "Ingl\u00E9s";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+            case "Espa\u00F1ol (Cuba)":
+              defaultLanguage = "English";
+              webDriverHelpers.selectFromCombobox(LANGUAGE_COMBOBOX, defaultLanguage);
+              break;
+          }
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+        });
+
+    When(
+        "I click on Data Dictionary hyperlink and download XLSX file in About directory",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(DATA_DICTIONARY_BUTTON);
+          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(
+              DATA_DICTIONARY_BUTTON, 50);
+          TimeUnit.SECONDS.sleep(3); // waiting for DATA_DICTIONARY_BUTTON
+        });
+
+    When(
+        "^I read data from downloaded XLSX file$",
+        () -> {
+          ReadXlsxFile();
+          TimeUnit.SECONDS.sleep(5); // waiting for xlsx file is read
+        });
+
+    When(
+        "I detect language for XLSX file content",
```

Comment:
here you say detect, but you also perform checks. Please rename it to refactor the purpose of the actions, and which language are we checking there?